boom! testing djpass-0.3.0 against nightly for nightly
boom! running: cargo +nightly build --frozen
boom! creating container for: cargo +nightly build --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly build --frozen cargobomb`
blam! 43f166a7ea73850dd4def1a4adeb217875cf53fb5c497c0b3443cda1bc47cf14
boom! running `docker start -a 43f166a7ea73850dd4def1a4adeb217875cf53fb5c497c0b3443cda1bc47cf14`
kablam!    Compiling djangohashers v0.2.6
kablam!    Compiling djpass v0.3.0 (file:///source)
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 3.88 secs
boom! running `docker rm -f 43f166a7ea73850dd4def1a4adeb217875cf53fb5c497c0b3443cda1bc47cf14`
blam! 43f166a7ea73850dd4def1a4adeb217875cf53fb5c497c0b3443cda1bc47cf14
boom! running: cargo +nightly test --frozen --no-run
boom! creating container for: cargo +nightly test --frozen --no-run
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly test --frozen --no-run cargobomb`
blam! 16d7cbd9f9a8720819de377556bb514fa281b12ad1a8bf319abce22dbae8be26
boom! running `docker start -a 16d7cbd9f9a8720819de377556bb514fa281b12ad1a8bf319abce22dbae8be26`
kablam!    Compiling djpass v0.3.0 (file:///source)
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 2.47 secs
boom! running `docker rm -f 16d7cbd9f9a8720819de377556bb514fa281b12ad1a8bf319abce22dbae8be26`
blam! 16d7cbd9f9a8720819de377556bb514fa281b12ad1a8bf319abce22dbae8be26
boom! running: cargo +nightly test --frozen
boom! creating container for: cargo +nightly test --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly test --frozen cargobomb`
blam! 2580ba21591f0e27cfda7af399b72217ff5f370755021b310811f5c02025f15a
boom! running `docker start -a 2580ba21591f0e27cfda7af399b72217ff5f370755021b310811f5c02025f15a`
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 0.0 secs
kablam!      Running /target/debug/deps/djpass-98945e6ac5480650
blam! 
blam! running 0 tests
blam! 
blam! test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured
blam! 
kablam!      Running /target/debug/deps/main-b43a60a6ecd61678
blam! 
blam! running 11 tests
blam! test test_bad_hash ... FAILED
blam! test test_argon2_algorithm ... FAILED
blam! test test_bad_algorithm ... FAILED
blam! test test_bad_password_and_hash_arguments ... FAILED
blam! test test_ignore_algoritm_when_verifying ... FAILED
blam! test test_good_password_and_hash_arguments ... FAILED
blam! test test_help ... FAILED
blam! test test_single_password_argument ... FAILED
blam! test test_password_and_algorithm_arguments ... FAILED
blam! test test_version ... FAILED
blam! test test_single_no_arguments ... FAILED
kablam! error: test failed
blam! 
blam! failures:
blam! 
blam! ---- test_bad_hash stdout ----
blam! 	thread 'test_bad_hash' panicked at 'called `Result::unwrap()` on an `Err` value: Error { repr: Os { code: 2, message: "No such file or directory" } }', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- test_argon2_algorithm stdout ----
blam! 	thread 'test_argon2_algorithm' panicked at 'called `Result::unwrap()` on an `Err` value: Error { repr: Os { code: 2, message: "No such file or directory" } }', /checkout/src/libcore/result.rs:859
blam! note: Run with `RUST_BACKTRACE=1` for a backtrace.
blam! 
blam! ---- test_bad_algorithm stdout ----
blam! 	thread 'test_bad_algorithm' panicked at 'called `Result::unwrap()` on an `Err` value: Error { repr: Os { code: 2, message: "No such file or directory" } }', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- test_bad_password_and_hash_arguments stdout ----
blam! 	thread 'test_bad_password_and_hash_arguments' panicked at 'called `Result::unwrap()` on an `Err` value: Error { repr: Os { code: 2, message: "No such file or directory" } }', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- test_ignore_algoritm_when_verifying stdout ----
blam! 	thread 'test_ignore_algoritm_when_verifying' panicked at 'called `Result::unwrap()` on an `Err` value: Error { repr: Os { code: 2, message: "No such file or directory" } }', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- test_good_password_and_hash_arguments stdout ----
blam! 	thread 'test_good_password_and_hash_arguments' panicked at 'called `Result::unwrap()` on an `Err` value: Error { repr: Os { code: 2, message: "No such file or directory" } }', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- test_help stdout ----
blam! 	thread 'test_help' panicked at 'called `Result::unwrap()` on an `Err` value: Error { repr: Os { code: 2, message: "No such file or directory" } }', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- test_single_password_argument stdout ----
blam! 	thread 'test_single_password_argument' panicked at 'called `Result::unwrap()` on an `Err` value: Error { repr: Os { code: 2, message: "No such file or directory" } }', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- test_password_and_algorithm_arguments stdout ----
blam! 	thread 'test_password_and_algorithm_arguments' panicked at 'called `Result::unwrap()` on an `Err` value: Error { repr: Os { code: 2, message: "No such file or directory" } }', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- test_version stdout ----
blam! 	thread 'test_version' panicked at 'called `Result::unwrap()` on an `Err` value: Error { repr: Os { code: 2, message: "No such file or directory" } }', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- test_single_no_arguments stdout ----
blam! 	thread 'test_single_no_arguments' panicked at 'called `Result::unwrap()` on an `Err` value: Error { repr: Os { code: 2, message: "No such file or directory" } }', /checkout/src/libcore/result.rs:859
blam! 
blam! 
blam! failures:
blam!     test_argon2_algorithm
blam!     test_bad_algorithm
blam!     test_bad_hash
blam!     test_bad_password_and_hash_arguments
blam!     test_good_password_and_hash_arguments
blam!     test_help
blam!     test_ignore_algoritm_when_verifying
blam!     test_password_and_algorithm_arguments
blam!     test_single_no_arguments
blam!     test_single_password_argument
blam!     test_version
blam! 
blam! test result: FAILED. 0 passed; 11 failed; 0 ignored; 0 measured
blam! 
boom! running `docker rm -f 2580ba21591f0e27cfda7af399b72217ff5f370755021b310811f5c02025f15a`
blam! 2580ba21591f0e27cfda7af399b72217ff5f370755021b310811f5c02025f15a
